vispWrapper/visp-demo.h

```diff
@@ -1,19 +1,20 @@
-// On Microsoft Windows, use dllexport to tag symbols.
-# if defined(_WIN32) || defined(__CYGWIN__)
-#   define VISP_WRAPPER_EXPORT __declspec(dllexport)
-# else // On Linux
-#   define VISP_WRAPPER_EXPORT
-# endif // defined(_WIN32) || defined(__CYGWIN__)
+#define TESTDLL_API __declspec(dllexport)

```

Comment:
I think that here `VISP_EXPORT` that is defined in `visp3/core/vpConfig` should be used instead of `VISP_WRAPPER_EXPORT`